### DISCUSSION TITLE
Bug Bug Bug

### DISCUSSION THREAD

**Discussion by wernerrvos** [six7/figma-tokens#1660] [General]
How can I remove your plugin footpring from all my components? 

I want to remove all of your plugin "footprints" from my components. How can i "remove all tokens" so they don't ... fuck up the rest of mt file?

Jesus, this has fucked me!

**Comment by wernerrvos** [comment]
Please just allow me to remove all tokens from my items. As in my program won't recognize the plugin anymore. And everything will magically turn back to normal!
Please jesus

**Comment by six7** [comment]
Just select all layers go to Inspect in the plugin, select all tokens and remove them.
